src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -437,6 +438,12 @@ private void CompileNamedType(NamedTypeSymbol containingType)
                                 hasStaticConstructor = true;
                             }
 
+                            // Replaced methods are not included in GetMembers().
+                            foreach (var replaced in method.GetReplacedMethods())
+                            {
+                                CompileMethod(replaced, extendedOrdinal, ref processedInitializers, synthesizedSubmissionFields, compilationState);
```

Comment:
The code gen might be more stable (e.g. for edit-and-continue scenarios) if the ordinal were per method rather than for the whole enclosing type.